gravitee-io/gravitee-access-management
# AM-766 SCIM group endpoint filter

## How To Test:

The easiest way would be to use existing Postman SCIM collection.

1. Follow the steps and create a couple users and groups.
2. Add users to a group [we have Patch group - add operation]
3. In the Postman `List` folder we have `List Groups` which lists all the groups.
4. Add a new GET request such as `{{gateway_url}}/{{domainHrid}}/scim/Groups?filter=displayName eq "Tour Guides"` should return only the mentioned group.

Please check with any JDBC as well if possible.
